### DISCUSSION TITLE
Enhanced Authentication Options for Pelican

### DISCUSSION THREAD

**Discussion by wolfun2001** [pelican-dev/panel#237] [Ideas]
We would like Pelican to offer more login methods, such as LDAP, OAuth2, or FIDO authentication. These new authentication methods would make Pelican more flexible and secure, allowing users to choose the login method that suits their needs and preferences. We hope the Pelican team will consider including these features in future versions.

**Comment by notAreYouScared** [comment]
Please Follow https://github.com/pelican-dev/panel/issues/385
